ddl/job_table.go

```diff
@@ -582,14 +595,13 @@ func GetBackfillJobsForOneEle(sess *session, batch int, excludedJobIDs []int64,
 
 	var err error
 	var bJobs []*BackfillJob
-	s := newSession(sess)
-	err = runInTxn(s, func(se *session) error {
-		currTime, err := GetOracleTimeWithStartTS(s)
+	err = runInTxn(sess, func(se *session) error {
+		currTime, err := GetOracleTimeWithStartTS(se)
```

Comment:
@zimulala I assume it should use the new session `se` from runInTxn instead of the outer `sess` here, or is it it intentional that it should have the timestamp from the outer `sess` session?

Reply:
Ditto.